inlong-sort/sort-connectors/elasticsearch-base/pom.xml

```diff
@@ -130,6 +130,20 @@
             <artifactId>audit-sdk</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.elasticsearch</groupId>
+            <artifactId>elasticsearch</artifactId>
+        </dependency>
```

Comment:
why add it. es-base module don't use es api

Reply:
fixed, please refresh the code to see the fix.